added atma.io solution + user

gse-atma-dev opened the pull request:
This update introduces our solution 'atma.io', laying the groundwork for the future inclusion of conformance tests.

raimundo-henriques commented on catalog/solutions/atma_io/2.0.1/solution.json:
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "atma_io",
```
```suggestion
  "id": "atma-io",
```

raimundo-henriques commented on catalog/users/atma_io.json:
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "atma_io",
```
```suggestion
  "id": "atma-io",
```

raimundo-henriques commented on catalog/solutions/atma_io/2.0.1/solution.json:
```diff
@@ -0,0 +1,8 @@
+  "name": "atma.io",
+  "version": "2.0.1",
+  "website": "https://atma.io/",
+  "provider": "atma_io",
```
```suggestion
  "provider": "atma-io",
```

raimundo-henriques reviewed (changes requested):
@gse-atma-dev Thank you for your submission!

Both the `solution-id` and the `user-id` must contain only letters, numbers, and dashes. 

Can you please replace the underscores (`_`) by dashes (`-`)?

Please see the inline comments.

cc. @prasadt1 
Thank you,

raimundo-henriques commented on catalog/users/atma_io.json:
```diff
@@ -0,0 +1,7 @@
+  "kind": "solutionprovider",
+  "name": "atma.io",
+  "email": "hello@atma.io",
+  "website": "https://www.atma.io/"
```
You might want to add the optional field

```
"logo": "<url-to-your-logo>"
```

That ensures that in your logo is displayed in the collaborators page.

I am sorry that the possibility of adding the "logo" field was not explicit in the documentation. I have updated it meanwhile.

raimundo-henriques commented on catalog/users/atma-io.json:
```diff
@@ -0,0 +1,7 @@
+  "kind": "solutionprovider",
+  "name": "atma.io",
+  "email": "hello@atma.io",
+  "website": "https://www.atma.io/"
```
You might want to add the optional field
```
"logo": "<url-to-your-logo>"
```
That ensures that in your logo is displayed in the collaborators page.

I am sorry that the possibility of adding the "logo" field was not explicit in the documentation. I have updated it meanwhile.

raimundo-henriques reviewed (commented):
@gse-atma-dev  Thank you for your submission!

We took the liberty to go ahead and fix some minor syntactical problems: since the use of underscores is not allowed by the catalog's syntax, we replaced them by dashes. 

Can you please review the changes?

Also, we strongly recommend adding a logo to your user `json`. Please see the inline comment.

Thank you,
Best,
cc @prasadt1

raimundo-henriques reviewed (commented):
@gse-atma-dev  Thank you for your submission!

We took the liberty to go ahead and fix some minor syntactical problems: since the use of underscores is not allowed by the catalog's syntax, we replaced them by dashes. 

Can you please review the changes?

Also, we strongly recommend adding a logo to your user `json`. Please see the inline comment.

Thank you,
Best,
cc @prasadt1

gse-atma-dev commented:
@raimundo-henriques Sorry for the delay and thank you for making the changes to the solution name.
As requested we added the logo and also added the results from the first conformance test with cirrus-nexus. 

Will add the result of the second compliance test (as soon as it finalized) via a second PR.

raimundo-henriques reviewed (changes requested):
Thank you very much for the changes. There are, however, a couple of issues that still need fixing (but which are easy to fix):
- The `"logo"` field belongs in the `atma-io.json` file, rather than on the `solution.json`. Please see my inline suggestions on how to fix these;
- The conformance test result json is missing a comma, please see my inline suggestion;
- The version of the Pathfinder Tech Specs tested must be either `"1.0.1"` or `"2.0.0"`, please see my inline suggestions.
Thank you for your cooperation!

gse-atma-dev commented:
hi, thanks for checking

I added your suggestions but the validation for the version in the pipeline seems now to fail again (I think it does not fail with version 2.0.1, see other PR from true carbon)

gse-atma-dev commented:
ah thank you, yes I think if everything is good from your side I think we can merge it

Thanks

gse-atma-dev commented:
The Vercel deployment action is failing, do we have to resolve this before merging the PR? can you merge it or do we have to do this?

raimundo-henriques commented:
Hello again @gse-atma-dev,
That's just because the `main` branch your branch is based on is already outdated, so Vercel's preview will break when trying to build it. This will be solved on merge, which will happen very soon!

Thank you!
